### DISCUSSION TITLE
difference between DGT versions

### DISCUSSION THREAD

**Discussion by riaadh** [EdNekebno/DGTCentaurMods#166] [General]
Hello there, 
I need your help before buying the DGT centaur (with the idea to install your mod :) ), in the documentation you say 

> This release only works on the second version of DGT centaur board * You own a first verion of the board if your board has visible black circles on the fields. If you decide to install this software anyway, you can use the web interface to display in real time what should be on the screen. The web interface allows you to safely power off the board too.

Do you have a serial number to check ? or all in the market are now second versions? 

Thanks a lot for your help and for this project

**Comment by wormstein** [comment]
Hi [riaadh](https://github.com/riaadh),
sorry for the late reply. August is a dead month, you know :)

Unfortunately, there is no way to distinguish between versions of the board by checking the serial number. The only way is to check for the black circles that are visible on the board. And, from what we know, a version 1 board will have those circles clearly visible.

If in doubt, you can always join us on our discord server (if not already) and if you have pics of the board, you can ask members for feedback on what version it might be. There are also sample pictures available that you could compare with.

Cheers,
-A

**Reply by riaadh** [reply]
thanks a lot for your help. I am totally new to rasberry so i will be grateful if you help me.
I can add all the informations in the wiki later.
